### PR TITLE
feature/RM-8036-Remove-InternetExplorer-Artifacts-From-WebTests

### PR DESCRIPTION
**Pull request by MichaelKetting**
https://re-motion.atlassian.net/browse/RM-8036

### CODE REVIEW

File: Build/Customizations/Server/Extensions.Server.targets [hunk 1]
```diff
@@ -70,7 +70,6 @@
   <Target Name="Server_CIBuild_Chrome" DependsOnTargets="TestBuild_ChromeConfiguration;CheckProperties;SetVersionCiAndNightlyBuild;TestBuild" />
```
Comment (iam-mholle) [inline Build/Customizations/Server/Extensions.Server.targets:70]: I found the following code elements that should be removed too:
- `Remotion.Web.ExecutionEngine.WxePageInfo.c_styleFileUrlForIE`
- `Remotion.Web.ExecutionEngine.WxePageInfo.s_styleFileKeyForIE`

File: Remotion/Web/Development.WebTesting/Utilities/DriverExtensions.cs [hunk 2]
```diff
@@ -48,7 +48,6 @@ public static string GetBrowserName ([NotNull] this IDriver driver)
           ChromeDriver _ => "Chrome",
           EdgeDriver _ => "Edge",
           FirefoxDriver _ => "Firefox",
-          InternetExplorerDriver _ => "InternetExplorer",
```
Comment (iam-mholle) [inline Remotion/Web/Development.WebTesting/Utilities/DriverExtensions.cs:51]: Using directive still present

File: Build/Customizations/Server/Extensions.Server.targets (hunk 1 above)
Comment (iam-mholle) [inline Build/Customizations/Server/Extensions.Server.targets:70]: `Remotion.Web.Development.WebTesting.CoypuElementScopeExtensions.IsTransparent` still has an IE workaround, TBD if it really only affects IE.

File: Remotion/Web/Development.WebTesting/WebTestConfigurationFactory.cs [hunk 3]
```diff
@@ -122,7 +122,7 @@ public IHostingConfiguration CreateHostingConfiguration ()
     }
 
     /// <summary>
-    /// Responsible for handling Browsers other than Chrome and Internet Explorer.
+    /// Responsible for handling Browsers other than Chrome, Edge, and Firefox.
```
Comment (iam-mholle) [inline Remotion/Web/Development.WebTesting/WebTestConfigurationFactory.cs:125]:
```suggestion
    /// Responsible for handling Browsers other than Chrome, Edge and Firefox.
```

**Review by iam-mholle — CHANGES_REQUESTED** [review]

File: Build/Customizations/Server/Extensions.Server.targets (hunk 1 above)
Comment (MichaelKetting) [reply Build/Customizations/Server/Extensions.Server.targets:70]: Those are out of scope for the current task (webtests only)

File: Remotion/Web/Development.WebTesting/WebTestConfigurationFactory.cs (hunk 3 above)
Comment (MichaelKetting) [reply Remotion/Web/Development.WebTesting/WebTestConfigurationFactory.cs:125]: Nope, starting with three options, English requires a comma before the `and`

File: Build/Customizations/Server/Extensions.Server.targets (hunk 1 above)
Comment (MichaelKetting) [reply Build/Customizations/Server/Extensions.Server.targets:70]: Good point. they would be equivalent. I'll update the docs but leave it in.

File: Remotion/Web/Development.WebTesting/WebTestConfigurationFactory.cs (hunk 3 above)
Comment (iam-mholle) [reply Remotion/Web/Development.WebTesting/WebTestConfigurationFactory.cs:125]: I didn't know the oxford comma is now mandatory in the english language. imo it just looks weird. Anyway, if it's required 🤷🏼‍♂️

Comment (MichaelKetting) [reply Remotion/Web/Development.WebTesting/WebTestConfigurationFactory.cs:125]: Interesting, didn't know it was only for them fancy talkers required. Anyhow, I think for me, it dates back to the prep lessons for my own masters thesis and I don't want to dig up whether it's actually required or just good practice for formal text. Either way, it's official enough to keep it when it's there, but may be okay to not mention it when it's missing during a review.

**Review by iam-mholle — APPROVED** [review]

**Pull request merged**